hack/generate/csv.sh

```diff
@@ -27,12 +27,14 @@ if [[ ${USE_RELEASE_NEXT:-} == "true" ]]; then
   knative_eventing_kafka_broker_images_release
   knative_backstage_plugins_images_release
   knative_serving_images_release
+  knative_kn_plugin_func_images_release
 else
   default_knative_eventing_images
   default_knative_eventing_istio_images
   default_knative_eventing_kafka_broker_images
   default_knative_backstage_plugins_images
   default_knative_serving_images
+  default_knative_kn_plugin_func_images
```

Comment:
Not sure on this one, if we simply should call `knative_kn_plugin_func_images` instead (and update knative_kn_plugin_func_images to not set any tags) or if keeping it this way, makes cleanup with the other functions more streamline as they are then all similar :shrug:

Reply:
I'm not sure I get it? PR looks good though